.github/workflows/cd.yml

```diff
@@ -190,63 +190,75 @@ jobs:
         uses: ffurrer2/extract-release-notes@v1
         with:
           release_notes_file: RELEASE_NOTES.md
+      - name: Prepend release notes
+        run: |
+          echo -e "mommy can also be installed using a package manager. [check the readme for more info](https://github.com/FWDekker/mommy/tree/${MOMMY_VERSION}#-installation)~\n" | cat - RELEASE_NOTES.md | tee RELEASE_NOTES.md
       - name: Checkout release
         uses: softprops/action-gh-release@v1
+        if: github.event_name != 'workflow_dispatch'
         with:
           draft: false
           prerelease: false
           tag_name: ${{ env.MOMMY_VERSION }}
           body_path: RELEASE_NOTES.md
           files: mommy*
+          discussion_category_name: announcements
 
   release-apt:
     needs: [ release-mommy ]
     runs-on: ubuntu-latest
 
     steps:
-      - name: Download built packages
-        uses: actions/download-artifact@v3
-        with:
-          name: dist
       - name: Checkout mommy
         uses: actions/checkout@v4
+        with:
+          path: src-mommy
```

Comment:
this is the crux of the fix. if you run `actions/checkout` and the target directory (which defaults to the `cwd`) is non-empty, it is made empty. so if we first download the built packages into `cwd` and then checkout into `cwd`, well, we just deleted the built packages. so instead, now we checkout into `src-mommy`~ (the fact that the checkout now happens before downloading the built packages is unrelated; that's just because i moved related steps closer to each other~)